src/Equinor.ProCoSys.IPO.Command/InvitationCommands/UpdateRfocAcceptedStatus/UpdateRfocAcceptedCommandHandler.cs

```diff
@@ -106,46 +104,44 @@ public async Task<Result<Unit>> Handle(UpdateRfocAcceptedCommand request, Cancel
             var commPkgNos = certificateCommPkgsModel.CommPkgs.Select(c => c.CommPkgNo).ToList();
             var mcPkgNos = certificateMcPkgsModel.McPkgs.Select(mc => mc.McPkgNo).ToList();
 
-            var mcPkgNosToUpdateStatusOn = await GetMcPkgNosToUpdateRfocStatusAsync(mcPkgNos, project);
-            var commPkgNosToUpdateStatusOn = await GetCommPkgNosToUpdateRfocStatusAsync(commPkgNos, project);
-            _invitationRepository.RfocAcceptedHandling(project.Name, commPkgNosToUpdateStatusOn, mcPkgNosToUpdateStatusOn);
-
-            try
-            {
-                await AddCertificateMcPkgRelationsAsync(request.ProCoSysGuid, mcPkgNos, project, cancellationToken);
-                await AddCertificateCommPkgRelationsAsync(request.ProCoSysGuid, commPkgNos, project, cancellationToken);
-            }
-            catch
+            var pcsMcPkgs = await _mcPkgApiService.GetMcPkgsByMcPkgNosAsync(project.Plant, project.Name, mcPkgNos);
+            var pcsCommPkgs = await _commPkgApiService.GetCommPkgsByCommPkgNosAsync(project.Plant, project.Name, commPkgNos);
```

Comment:
I feel this should return an empty list, and never null? would simplify IF statement under, and we can do the select here

Reply:
Removing null check.

I am not doing the select here, because I want to store a certificate relation to a commpkg even if the whole commpkg does not have handover status = "Accepted". We can discuss this further if you want.